src/Factory.ts

```diff
@@ -7,8 +8,11 @@ import {ExpressServer} from "./server/ExpressServer";
  * @param expressApp Express application instance
  * @param requireDirs Array of directories where from controller files will be loaded
  */
-export function registerActionsInExpressApp(expressApp: any, requireDirs?: string[]): ControllerRunner {
+export function registerActionsInExpressApp(expressApp: any, requireDirs?: string[], container?: Container): ControllerRunner {
```

Comment:
I would like to accept this changes for now. But in the future this and some other parts will be changed anyway since the library is in early beta, I just want to warn you because you are one of the few library users (I hope you are okay with changes)